prisma-fmt/tests/code_actions/scenarios/relation_mode_prisma_missing_index/diagnostics.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "range": {
+            "start": {
+                "line": 19,
+                "character": 20
+            },
+            "end": {
+                "line": 19,
+                "character": 63
+            }
+        },
+        "severity": 1,
+        "message": "With `relationMode = \"prisma\"`, no foreign keys are used, so relation fields will not benefit from the index usually created by the relational database under the hood. This can lead to poor performance when querying these fields. We recommend adding an index manually. Learn more at https://pris.ly/d/relation-mode#indexes"
```

Comment:
I think the message was updated in the meantime, but as this is just input and expectation at the same time (And does not use the actual correct warning), this should be fine.

(Or should this actually be wired up to the real warning?)